privacyidea/lib/tokenclass.py

```diff
@@ -132,6 +132,11 @@ class TOKENMODE(object):
     OUTOFBAND = 'outofband'
 
 
+class ROLLOUTSTATE(object):
+    CLIENTWAIT = 'clientwait'
+    PENDING = 'pending'  # TODO: to be used, if the server needs to take action
```

Comment:
Why is there a `TODO`? because it is currently not used? But this type will stay or will it change in the future?

Reply:
And what about a `finished` state (instead of `""`)? or is that too much clutter in the token class?

Reply:
I was aslo thinking about a state "enrolled". 
But then we could have tokens, that are "enrolled", since they are in the `rollout_state` "enrolled", but there are also tokens that are "enrolled", since these token types never used the `rollout_state`.
So I did not want to overcomplicate it is the first step.

But I will remove the "TODO".

Reply:
Done in 0780585ec89240821dd72594f9aef4345fc4600c